scrapy/downloadermiddlewares/httpproxy.py

```diff
@@ -26,7 +26,7 @@ def _get_proxy(self, url, orig_type):
         proxy_url = urlunparse((proxy_type or orig_type, hostport, '', '', '', ''))
 
         if user:
-            user_pass = '%s:%s' % (unquote(user), unquote(password))
+            user_pass = to_bytes('%s:%s' % (unquote(user), unquote(password)))
```

Comment:
@kmike [Here is explained](http://stackoverflow.com/questions/4400678/http-header-should-use-what-character-encoding), there is also a link to the source.